app/assets/stylesheets/views/_landing_page/main-navigation.scss

```diff
@@ -24,17 +24,17 @@
   @include govuk-font(19);
   background: none;
   border: 0;
-  padding-left: govuk-spacing(2);
-  padding-right: govuk-spacing(2);
+  padding-left: govuk-spacing(3);
+  padding-right: govuk-spacing(3);
+  margin-left: -15px; // Ensures the button looks perfectly left-aligned with content.
```

Comment:
Nitpick - would it be possible to use `govuk-spacing(-3)` here for consistency?

Reply:
@JamesCGDS Will do - I didn't realise we could use negative values, thanks 👍